site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -257,6 +257,37 @@ paths:
             403:
                $ref: '#/components/responses/NotAllowed'
 
+      get:
+         tags:
+            - devices
+         summary: Retrieve all devices for a tenant
+         operationId: getAllDevicesForTenant
+         requestBody:
+            description: Filters, paging and sorting options.
+            content:
+               application/json:
+                  schema:
+                     $ref: '#/components/schemas/PagingSortingFilteringOptions'
+            required: false
+         responses:
+            200:
+               description: operation successful
+               content:
+                  application/json:
+                     schema:
+                        $ref: '#/components/schemas/Device'
+               headers:
+                  ETag:
+                     description: Version of the resource
+                     schema:
+                        type: string
+            400:
+               $ref: '#/components/responses/MalformedRequest'
+            401:
+               $ref: '#/components/responses/Unauthorized'
+            404:
```

Comment:
I am not sure what "not found" would mean in this case. When searching, I would expect an empty result would still be "200 OK".

Reply:
and what if the tenant doesn't exist ?